examples/dashboards/dashboards.md

```diff
@@ -0,0 +1,18 @@
+# Dashboards
+
+There exists Grafana dashboards for each component (not all of them complete) targeted for environments running Kubernetes:
+
+- [Thanos Overview](thanos-overview.json)
+- [Thanos Compact](thanos-compact.json)
```

Comment:
Is this valid? (: What we choosen at the end? Maybe worth to move to
`Querier`, Compactor`, `Receiver` etc (store stays for now)

As per: https://github.com/thanos-io/thanos/issues/1871

Reply:
🤦‍♂ I have done exactly the opposite. I stuck with the command names. This not big of a change I can do it as it suggested in the issue.